net/scripts/gce-provider.sh

```diff
@@ -162,6 +162,12 @@ cloud_CreateInstances() {
   # shellcheck disable=SC2206 # Do not want to quote $imageName as it may contain extra args
   args+=(--image $imageName)
 
+  # shellcheck disable=SC2068 # Do not want to quote $machineType
+  for word in ${machineType[@]}; do
```

Comment:
machinetype isn't an array...

Reply:
nice localizing of the nasty

Reply:
> machineType isn't an array...

lol but it totally works still.  ❤️ bash.